weblate/templates/trans/alert/duplicatelanguage.html

```diff
@@ -9,11 +9,11 @@
 
     <ul>
     {% if not analysis.monolingual %}
-      <li>{% trans "You might be using monolingual translation files as bilingual. In that case please configure the monolingual base language file." %} {% documentation_icon "formats" "bimono" %}</li>
+      <li>{% trans "You might be using a monolingual translation files as a bilingual one. If that is the case, please set the monolingual base language file origin." %} {% documentation_icon "formats" "bimono" %}</li>
```

Comment:
Can you please clarify why origin?

Reply:
It is already "set" in a sense, it just needs pointing out.
Same as "configuring it" could also mean configuring the file itself.

Reply:
It's about filling in https://docs.weblate.org/en/latest/admin/projects.html#monolingual-base-language-file in the component configuration.